server/cp/ai/semaphore.py

```diff
@@ -70,49 +38,68 @@ class Semaphore(AIServiceBase):
     name = "semaphore"
     label = "Semaphore autotagging service"
 
-    def __init__(self, app):
+    def __init__(self, data):

```

Comment:
afaik there is no data passed on init, it's the application object

Reply:
Resolved